service/init.go

```diff
@@ -18,9 +18,9 @@ import (
 	"syscall"
 	"time"
 
+	grpcClient "github.com/5GC-DEV/config5g-cdac/proto/client"
```

Comment:
We will have to port this change once other big change from Canonical team merges.